website/documentation/reference.py

```diff
@@ -19,34 +19,53 @@ def generate_class_doc(class_obj: type, part_title: str) -> None:
 
     mro = [base for base in class_obj.__mro__ if base.__module__.startswith('nicegui.')]
     ancestors = mro[1:]
-    attributes = {}
+    attributes: dict[str, tuple[type, object | None]] = {}
     for base in reversed(mro):
         for name in dir(base):
             if not name.startswith('_') and _is_method_or_property(base, name):
-                attributes[name] = getattr(base, name, None)
-    properties = {name: attribute for name, attribute in attributes.items() if not callable(attribute)}
-    methods = {name: attribute for name, attribute in attributes.items() if callable(attribute)}
+                attributes[name] = (base, getattr(base, name, None))
+    properties = {name: (ancestor, attribute) for name, (ancestor, attribute)
+                  in attributes.items() if not callable(attribute)}
+    methods = {name: (ancestor, attribute) for name, (ancestor, attribute) in attributes.items() if callable(attribute)}
+
+    def render_section(items: dict[str, tuple[type, object | None]], is_method: bool) -> None:
+        sorted_items = sorted(items.items())
+        native = [(n, o, a) for n, (o, a) in sorted_items if o is class_obj]
+        inherited_items = [(n, o, a) for n, (o, a) in sorted_items if o is not class_obj]
+
+        def render_item(name: str, owner: type, attr: object | None) -> None:
+            if is_method:
+                decorator = ''
+                owner_attr = owner.__dict__.get(name)
+                if isinstance(owner_attr, staticmethod):
+                    decorator += '`@staticmethod`<br />'
+                if isinstance(owner_attr, classmethod):
+                    decorator += '`@classmethod`<br />'
+                ui.markdown(f'{decorator}**`{name}`**`{_generate_method_signature_description(attr)}`') \
+                    .classes('w-full overflow-x-auto')
+            else:
+                ui.markdown(f'**`{name}`**`{_generate_property_signature_description(attr)}`')
+            docstring = getattr(attr, '__doc__', None)
+            if attr is not None and docstring:
+                _render_docstring(docstring).classes('ml-8')
+
+        for name, owner, attr in native:
+            render_item(name, owner, attr)
+        if inherited_items:
+            with ui.row().classes('items-center gap-2 text-sm text-gray-500 dark:text-gray-400 mt-2'):
+                ui.icon('sym_o_account_tree')
```

Comment:
```suggestion
                ui.icon('account_tree')
```

Because of https://github.com/zauberzeug/nicegui/pull/5515

It's available: https://fonts.google.com/icons?icon.query=account_tree&icon.size=24&icon.color=%231f1f1f&icon.set=Material+Icons